release_planning.rb

```diff
@@ -2,12 +2,14 @@
 
 require 'optparse'
 require_relative 'octokit_utils'
+require_relative 'release_planning/pupet_module'
```

Comment:
sorry to be a debbie downer, the javascript plugins will not work in jenkins. due to the over zealous lock down there.